### ISSUE TITLE
Document current validation script

### ISSUE THREAD

**Issue by AndrewEdmonds11** [Mu2e/TrkAna#23]
Long term, we should have some form of code validation. I'm not sure what this would look like (we probably don't need to be as rigorous as Offline) but this is low priority.

**Comment by sophiemiddleton** [comment]
Should we discuss this in one of our developers meetings at some point?

**Comment by kutschke** [comment]
Is this the same issue discussed in: https://github.com/Mu2e/CI/issues/122  ?  Or is it different in some way?

**Comment by sophiemiddleton** [comment]
I think its separate, but @AndrewEdmonds11 would know more

**Comment by AndrewEdmonds11** [comment]
So I opened this a while ago, and recently opened Mu2e/CI#122 to determine whether we want something in the CI (we landed on it being unnecessary)

For this issue, I'll update it to be that I document how to run the validation script that I wrote.

The other part of validation that we need to keep in mind for the future is to make sure the utility classes etc. work. Maybe this will be ocvered when RefAna starts running in the nightly so we can revisit then
